docs/src/pages/quasar-cli-vite/state-management-with-pinia.md

```diff
@@ -65,26 +65,60 @@ We've created the new Pinia store, but we haven't yet used it in our app. In a V
 
 ```html
 <template>
-  <div>
-    <q-toggle v-model="store.counter" />
+  <div class="example">
```

Comment:
This is not good for brevity. Since this is not an interactive example we can omit the class to make the code simpler.
```suggestion
  <div>
```